.github/workflows/tools.yml

```diff
@@ -154,7 +154,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/scoring
     - name: Configuration Scorer Tests
-      if: steps.changes.outputs.scoring == 'true'
+      if: steps.changes.outputs.scoring == 'true' OR steps.changes.outputs.instance_validator == 'true'
```

Comment:
the or is not supported, please use the expression here: https://docs.github.com/en/actions/learn-github-actions/expressions#operators

Reply:
Done.